code/modules/holiday/holidays.dm

```diff
@@ -421,11 +487,30 @@
 /datum/holiday/pride/getStationPrefix()
 	return pick("Pride", "Gay", "Bi", "Trans", "Lesbian", "Ace", "Aro", "Agender", pick("Enby", "Enbie"), "Pan", "Intersex", "Demi", "Poly", "Closeted", "Genderfluid")
 
+/datum/holiday/stonewall
```

Comment:
Riot Anniversaries are a bad idea, do to all the other ones out their making it harder and harder to get them all in a list and have a somewhat non-always holiday station

Reply:
This specific event has a very strong significance in the context of the LGBTQIA+ community. It was a pretty huge turning point for LGBTQIA+ rights. It's the same reason why some people call it the Stonewall Uprising instead of the Stonewall Riots.

Reply:
Yeah, it's a major deal and I think it's perfectly fine to include.

Reply:
I just think it sets a to low bar to add in more holidays after riots.

Reply:
imo the holidays should be international, and have deeper history.
IC its like 2552 and I just dont think many people would really know about it.
But again those are my thoughts on it, after all we have Thx Giving/4th of July in so 🤷

Reply:
Holidays generally stray towards the realm of OOC in quite a few instances. There's a holiday in the code for SS13's birthday, for instance, which gives factoids about it's age at each major milestone. There's also Programmers' day, which replaces the station's name prefix with generic code bits. The only one really straying towards being IC are the ones with ingame effects, such as Christmas and Valentine's day.

Also, the majority of the holidays this PR adds do indeed have deeper history behind them. The Stonewall Riots, for instance, is what sparked the modern-day LGBT rights movements all around the globe. Here in the US, being gay used to be an actual crime, to the point where police regularly raided gay bars and arrested patrons for simply being present. Stonewall was the final straw that ended up sparking large-scale protests, and eventually lead to LGBT individuals being granted basic rights to live without their mere existence being a crime. This is also why a lot of people repeatedly stress: "No cops at pride". The anniversary of the riots is a US-centric date, sure, but it carries some pretty huge significance due to the effect it had on civil rights here in the US.

Reply:
I understand that, I maybe just wish their was more of an in game affect for them all and maybe some "fake" holidays as well for lore and such.